### DISCUSSION TITLE
[Proposal]: change theme config boxShadow to array syntax

### DISCUSSION THREAD

**Discussion by blowsie** [tailwindlabs/tailwindcss#1170] [Ideas]
Hey.

Today I wrote [some code](https://gist.github.com/blowsie/4727cb949b13e5338bee64dc8f2cf5ca) which converts a tailwind theme config into a `_variables.scss` file.

**Why?**
So that the variables can be used to pass into other ui-frameworks such as bootstrap or vuetify.
Doing this, in my opinion has huge potential to apply the design system to functional components offered by such frameworks, or even just as an iterim solution whilst migrating to tailwind.

Everything worked fine, except [`box-shadow`](https://github.com/tailwindcss/tailwindcss/blob/bcc2449a29082da9868dbd2b03d332c1633d432c/stubs/defaultConfig.stub.js#L185), i dug into the code and found this is because the output is not valid scss map syntax.

[`font-family`](https://github.com/tailwindcss/tailwindcss/blob/bcc2449a29082da9868dbd2b03d332c1633d432c/stubs/defaultConfig.stub.js#L222) , however which has the same desired output works fine.
This is because it is using arrays.

**What?**
I propose that `box-shadow` & `font-family` use the same array syntax for consistency and so that they can be exported as above a little easier.

If accepted, I would be happy to try and create a PR. 

Thanks!

**Comment by hacknug** [comment]
I think it would make sense as long as both syntaxes are supported.

**Reply by TrentFoulds** [reply]
It looks like it expects your Theme to have at least 25 shadows, in order to create the progression seen in Material UI. While I certainly don't recommend removing the shadows if you're trying to follow Material UI standards, one easy way to do it might just be to set all levels of elevation to none.  [PMP Payslip](https://www.pmppayslip.net/)
